server/amp/handlers/visual-stories-bookend.js

```diff
@@ -0,0 +1,75 @@
+const get = require("lodash/get");
+
+function getImageUrl(story, config) {
+  if (get(story, ["story-template"]) === "news-elsewhere") {
+    return get(story, ["metadata", "reference-url"], "");
+  }
+  return `${config["sketches-host"]}/${story.slug}`;
+}
+
+async function bookendHandler(req, res, next, { config, client }) {
+  const relatedStoriesResponse = await client.getRelatedStories(
+    req.query.storyId,
+    req.query.sectionId
```

Comment:
what happens when either one or both of the query params is not received?

Reply:
adding a check for this case